tests/phases/design/test_mooring_system_design.py

```diff
@@ -83,3 +167,30 @@ def test_custom_num_lines():
     m.run()
 
     assert m.design_result["mooring_system"]["num_lines"] == 5
+
+
+def test_new_old_semitaut_mooring_system():
```

Comment:
Just to circle back from way up in the review, we should delete this if we're going to adopt the multi-option mooring design class (preferred). Unless there's opposition from elsewhere, I'm not sure it's wise to introduce two ways to run the same exact logic, when one is adding a whole new design module.